boards/arm/bcm958402m2_m7/bcm958402m2_m7_defconfig

```diff
@@ -18,3 +18,9 @@ CONFIG_UART_NS16550=y
 # Enable Console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+# Enable PAXDMA
+CONFIG_DMA=y
+CONFIG_DMA_64BIT=y
+CONFIG_DMA_IPROC_PAX_V2=y
+CONFIG_DMA_0_NAME="pax_dma"
```

Comment:
remove CONFIG_DMA_0_NAME

Reply:
@galak: dts/arm/broadcom/viper-common.dtsi   contains paxdma node definition as,

paxdma: paxdma@4e100800 {
                compatible = "brcm,iproc-pax-dma-v2";
                label = "pax_dma";
                .................................
};

So CONFIG_DMA_0_NAME is set in config file to "pax_dma", otherwise I need to use "DMA_0" for pl330 and "DMA_1" for paxdma.

Let me push change in dts for setting label as "DMA_0" for pl330 and "DMA_1" for PAXDMA.

That way we can eliminate setting the names explicitly.
I think naming

Reply:
Sounds good.